bin/cli-flags.js

```diff
@@ -123,6 +123,41 @@ module.exports = {
         delete opts.clientOverlay;
       },
     },
+    {
+      name: 'client-need-client-entry',
```

Comment:
Maybe we need a better name, double `client` can be confusing.

Reply:
Yep, good catch I will think about it

Reply:
Maybe: 

```js
client: {
   appendDefaultEntries: true,
   appendHotEntries: true
}
```

Reply:
hm, give me time to think about these options, maybe we need `{ client: (entry) => { return clientOption; } }`, so you will setup hot/client entry for multiple entrypoints